### PR TITLE
Better docstring for the output of Matrix.rref()

### LINKED ISSUE #11597: rref() should return a Matrix
Currently calling `.rref()` on a matrix produces a tuple with a matrix and a list of pivots.  I think this is unexpected for new users and the default should be that `rref()` returns the row-reduced matrix only.  An `include_pivots` flag could be added to get the current behavior where a tuple is returned.

### PR DESCRIPTION
**Pull request by nikoskaragiannakis**
Fixes #11597 

A small addition to the docstring of rref() in order to explain clearly how to grab the echelon form of a matrix and the pivots separately.

### CODE REVIEW

File: sympy/matrices/matrices.py [hunk 1]
```diff
@@ -4083,6 +4083,13 @@ def rref(self, iszerofunc=_iszero, simplify=False):
         (Matrix([
         [1, 0],
         [0, 1]]), [0, 1])
+        >>> m_echelon, e_pivots = m.rref()
```
Comment (aktech) [inline sympy/matrices/matrices.py]: Lets rather use `rref_matrix`, `rref_pivots`

**Review by aktech — COMMENTED** [review]
Minor nitpick

File: sympy/matrices/matrices.py (hunk 1 above)
Comment (nikoskaragiannakis) [inline sympy/matrices/matrices.py]: done :+1:

**Review by aktech — APPROVED** [review]
Looks good to me.

**Pull request merged**